test/runtests.jl

```diff
@@ -274,11 +274,11 @@ end
 import Conda
 if PyCall.conda
     # install the tiny conda-api package as a test
```

Comment:
comment needs to be changed: `# import pyzmq to test PR #294`.

Will importing `pyzmq` cause problems if we run `Pkg.test("PyCall")` inside IJulia or in any other context where `ZMQ.jl` has already loaded and initialized `libzmq`?